File: level 55/home/codewars.py
# task 1
def small_enough(array, limit):
    if max(array) <= limit:
        return True
    else:
        return False
    
# task 2

def sum_two_smallest_numbers(numbers):
    sorted1 = sorted(numbers)
    sum = sorted1[0] + sorted1[1]
    return sum

# task 3

def is_square(n):
    if n < 0:
        return False
    square_root = n ** 0.5
    
    num = int(square_root)
    
    if num*num == n:
        return True
    else:
        return False
    
# task 4

def password(st):
    upper = False
    lower = False
    number = False
    if st == '':
        return False
    for i, char in enumerate(st):
        if char.isupper():
            upper = True
        elif char.islower():
            lower = True
        elif char.isnumeric():
            number = True
    if i >= 7 and (upper == True) and (lower == True) and (number == True):
        return True
    return False

# task 5

def get_size(w,h,d):
    area = w * h * d
    volume = (d*w*2) + (d*h*2) + (w*h*2)
    both = [volume, area]
    return both

File: level 70/class/codewars.js
function sumTwoSmallestNumbers(numbers) {
  let sum = 0;
  
  numbers.sort(function(a, b){return a - b})
  
  sum += numbers[0] + numbers[1];
  
  return sum;
}

var isSquare = function(n){
  let sQ = Math.sqrt(n);
  
  if(sQ === Math.floor(sQ)){
    return true;
  }else{
    return false;
  }
}

function replace(s) {
  let res = '';
  
  for(let i = 0; i < s.length; i++){
    let char = s[i];
    
    if(char === 'a' || char === 'e' || char === 'i' || char === "o" || char === "u" || char === "A" || char === 'E' || char === 'I' || char === "O" || char === "U"){
      res += '!';
    }else{
      res += char;
    }
  }
  return res

}
